src/runner.js

```diff
@@ -14,8 +14,6 @@ async function run() {
   const startDate = new Date()
   const packageJson = await getPackageJson()
   const logger = getLogger()
-  logger.info(`${packageJson.name} started`)
```

Comment:
I find it useful to see that the job could start successfully, don't you think?

Reply:
https://github.com/commercetools/commercetools-subscriptions/pull/33#discussion_r858493977